testing/test_driver.py

```diff
@@ -107,6 +107,37 @@ def test_pass(driver_kwargs):
     testdir.quick_qa("--driver", "Remote", file_test, passed=1)
 
 
+@pytest.mark.parametrize(
+    ("host", "expected"),
+    [
+        ("https://some.host.com", "https://some.host.com"),
+        ("http://some.host.com", "http://some.host.com"),
+        ("some.host.com", "http://some.host.com"),
+    ],
+)
+def test_host_protocol(host, expected, testdir):
+    filetest = testdir.makepyfile(
+        """
+        import pytest
+        @pytest.mark.nondestructive
+        def test_pass(driver_kwargs):
+            assert driver_kwargs['command_executor'] == '{}:4444/wd/hub'
+        """.format(
+            expected
+        )
+    )
+    testdir.quick_qa(
+        "--driver",
+        "Remote",
+        "--selenium-host",
+        host,
+        "--selenium-port",
```

Comment:
Since 4444 is the default there is no need to explicitly pass it here but fine for me.